tgui/packages/tgui/interfaces/PreferencesMenu/QuirksPage.tsx

```diff
@@ -201,6 +202,21 @@ export const QuirksPage = (props, context) => {
                 return `This is incompatible with ${incompatibleQuirk}!`;
               }
             }
+
+            for (const speciesblacklist of speciesquirkBlacklist) {
+              if (speciesblacklist.indexOf(quirk.name) === -1) {
+                continue;
+              }
+
+              for (const speciesquirkBlacklist of speciesblacklist) {
+                if (
+                  speciesquirkBlacklist !== quirk.name &&
+                  selectedQuirkNames.indexOf(speciesquirkBlacklist) !== -1
+                ) {
+                  return `This is incompatible with ${speciesquirkBlacklist}!`;
+                }
+              }
+            }
```

Comment:
Remove this entire section. It's not correctly implemented, and does not belong in this conditional branch anyways.